simple/bug_world/bug.py

```diff
@@ -62,10 +60,12 @@ def get_data_dict(self, step):
         "step": step,
       }
 
-
   def is_too_thin(self):
     """ bug will die if it's too think """
-    return self.size / self.age < self._MIN_SIZE_RATIO
+    if self.age == 0:
+      return False
+    MIN_SIZE_RATIO = 0.5
```

Comment:
can you change the value to 0.1? I tried yesterday and found 0.5 is too small that bug grows too slow to keep this ratio.